src/ol_infrastructure/applications/concourse/iam_policies/infra.py

```diff
@@ -12,6 +12,8 @@
         {
             "Effect": "Allow",
             "Action": [
+                "autoscaling:DescribeAutoScalingGroups",
+                "autoscaling:DescribeLaunchConfigurations",
```

Comment:
We should probably split the permissions into different statements so we can track which permissions are needed for what purposes. It might get unwieldy, but it's probably worth a good college try.